plugins/block-dynamic-connection/test/index.js

```diff
@@ -25,8 +25,61 @@ function createWorkspace(blocklyDiv, options) {
 document.addEventListener('DOMContentLoaded', function() {
   BlockDynamicConnection.overrideOldBlockDefinitions();
 
+  const toolbox = {
+    kind: 'flyoutToolbox',
+    contents: [
+      {
+        kind: 'block',
+        type: 'text_join',
+      },
+      {
+        kind: 'block',
+        type: 'lists_create_with',
+      },
+      {
+        kind: 'block',
+        type: 'controls_if',
+      },
+      {
+        kind: 'block',
+        type: 'logic_boolean',
+        fields: {
+          BOOL: 'TRUE',
+        },
+      },
+      {
+        kind: 'block',
+        type: 'text',
+        fields: {
+          TEXT: 'abc',
+        },
+      },
+      {
+        kind: 'block',
+        type: 'math_number',
+        fields: {
+          NUM: '123',
+        },
+      },
+      {
+        kind: 'block',
+        type: 'text_print',
+        inputs: {
+          TEXT: {
+            shadow: {
+              type: 'text',
+              fields: {
+                TEXT: 'abc',
+              },
+            },
+          },
+        },
+      },
+    ],
+  };
+ 
   const defaultOptions = {
-    toolbox: document.getElementById('toolbox'),
+    toolbox
```

Comment:
@uyamini I guess you should also remove the XML block from the `index.html`. But maybe one of the maintainers can clarify on that. 
  
 Additionally, You should add a comma after the toolbox.

Reply:
Yes! Removing the XML is much appreciated =)